tasks/migrations/0002_auto_20200509_1255.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.0.5 on 2020-05-09 12:55
+
+from django.db import migrations, models
+
```

Comment:
Please explain why this migrations is need for this PR.

Reply:
They aren't related to Registration View, I think these migrations were to be done in the Tasks app probably when you suggested some changes but maybe got missed and so when I pulled from dev and ran migrations this happened to get created and was added alongside rest of the additions as I added all 😅 

Shall I delete this file and commit?

Reply:
Yup please delete this file and commit.